components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/test/java/org/wso2/carbon/apimgt/rest/api/store/impl/PetStoreSwaggerTestCase.java

```diff
@@ -0,0 +1,2071 @@
+package org.wso2.carbon.apimgt.rest.api.store.impl;
+
+public class PetStoreSwaggerTestCase {
+
+    public static final String CORRECT_LANGUAGE ="java";
+    public static final String INCORRECT_LANGUAGE ="java2";
+    public static final String SWAGGER_PET_STORE_CORRECT="{\n" +
```

Comment:
Let's use resource files in here as well. Also please add the license headers. Check all the places in the PR.